src/mouse.cpp

```diff
@@ -1136,7 +1136,7 @@ void GraphicsWindow::MouseLeftDown(double mx, double my) {
                     AddToPending(hr);
                     Request *r = SK.GetRequest(hr);
                     r->str = "Abc";
-                    r->font = "arial.ttf";
+                    r->font = "BitstreamVeraSans-Roman-builtin.ttf";
```

Comment:
This isn't updated to use the `res://`, and the `-builtin` suffix is no longer necessary to disambiguate.

Reply:
The res:// is added to the `fontFile` when loading the fonts. Setting a font is only done by the base name, which does not contain the res://. This is also why I would keep the builtin suffix.